deltachat-ios/AppDelegate.swift

```diff
@@ -49,13 +51,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     // when the app wakes up from "suspended" state
     // (app is in memory in the background but no code is executed, IO stopped)
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        pendingAppInitialization = true;
```

Comment:
```suggestion
        pendingAppInitialization = true
```

superfluous semicolon, also, there seems to be no need to set it here.

Reply:
java routine....